Api.Client/Model/PagingInfo.cs

```diff
@@ -0,0 +1,24 @@
+namespace Nexosis.Api.Client.Model
+{
+    /// <summary>
+    /// Used to request certain paging in a response
+    /// </summary>
+    public class PagingInfo
```

Comment:
Here I think we need to continue to have a default rather than having the direct result of a list be the max page size.

Reply:
Done